roles/openshift_logging/tasks/install_elasticsearch.yaml

```diff
@@ -17,13 +17,13 @@
 
 # This should be used to generate new DC names if necessary
 - name: Create new DeploymentConfig names for Elasticsearch
-  set_fact: es_dc_pool={{es_dc_pool + [deploy_name]}}
+  set_fact: es_dc_pool={{es_dc_pool|default([]) + [deploy_name]}}
   vars:
     component: es
     es_cluster_name: "{{component}}"
     deploy_name_prefix: "logging-{{component}}"
     deploy_name: "{{deploy_name_prefix}}-{{'abcdefghijklmnopqrstuvwxyz0123456789'|random_word(8)}}"
-  with_sequence: count={{ openshift_logging_current_es_size | int - openshift_logging_es_cluster_size | int }}
+  with_sequence: count={{ (openshift_logging_current_es_size | int - openshift_logging_es_cluster_size | int) |abs}}
```

Comment:
can we flip the order of the subtraction instead of using a filter? we should be failing out if openshift_logging_es_cluster_size < the current cluster size